mesonbuild/dependencies/base.py

```diff
@@ -31,6 +32,8 @@
     MesonException, Popen_safe, version_compare_many, version_compare, listify
 )
 
+#from .dub import DubDependency
```

Comment:
__[Flake8]__

[E265] block comment should start with '# '


<!-- Comment by [SideCI](https://sideci.com) -->